codemp/rd-rend2/glsl/lightall.glsl

```diff
@@ -532,17 +533,19 @@ vec3 CalcIBLContribution(
 #if defined(USE_CUBEMAP)
 	// EnvironmentBRDF is supposed to work with gloss. Invert roughness for now, but this needs
 	// to be replaced at some point with preconvoluted cubemap code.
-	vec3 reflectance = EnvironmentBRDF(1.0 - roughness, NE, specular);
+	//vec3 reflectance = EnvironmentBRDF(1.0 - roughness, NE, specular);
```

Comment:
Please don't leave commented code here :) We can always fetch it back from git later on if we need it.